proxyclient/m1n1/hv/__init__.py

```diff
@@ -1562,10 +1562,12 @@ def cpustart_wh(base, off, data, width):
             chip_id = self.u.adt["/chosen"].chip_id
             if chip_id in (0x8103, 0x6000, 0x6001, 0x6002):
                 cpu_start = 0x54000 + die * 0x20_0000_0000
-            elif chip_id in (0x8112,):
+            elif chip_id in (0x8112, 0x8122):
                 cpu_start = 0x34000 + die * 0x20_0000_0000
             elif chip_id in (0x6020, 0x6021, 0x6022):
                 cpu_start = 0x28000 + die * 0x20_0000_0000
+            elif chip_id in (0x6030, 0x6031, 0x6034,):
+                cpu_start = 0x88000 + die * 0x20_0000_0000
```

Comment:
this is incorrect, Pro has a different CPU start offset than Max. I'd suggest leaving T6030 out of this PR